cli/flags.rs

```diff
@@ -309,7 +310,7 @@ fn clap_root<'a, 'b>() -> App<'a, 'b> {
     // Disable clap's auto-detection of terminal width
     .set_term_width(0)
     // Disable each subcommand having its own version.
-    .version(crate::version::DENO)
+    .version(version)
     .long_version(LONG_VERSION.as_str())
```

Comment:
After another thought... maybe add `SHORT_VERSION` lazy static similar to `LONG_VERSION`?